packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -726,7 +726,7 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
     async function createNftHTSToken(account) {
       const mainContract = new ethers.Contract(mainContractAddress, TokenCreateJson.abi, accounts[0].wallet);
       const tx = await mainContract.createNonFungibleTokenPublic(account.wallet.address, {
-        value: BigInt('10000000000000000000'),
+        value: BigInt('30000000000000000000'),
```

Comment:
This was included in this PR https://github.com/hashgraph/hedera-json-rpc-relay/pull/2972 right?

Reply:
Yes